414.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var thirdMax = function (nums) {
+  const sorted = [...nums].sort((a, b) => a - b);
```

Comment:
You can create an array of distinct numbers with `Set` like `[... new Set(1, 1, 3, 4)]`, which is helpful to manipulate distinct elements.